core/generated-sources/src/main/kotlin/org/jetbrains/kotlinx/dataframe/documentation/ColumnExpression.kt

```diff
@@ -0,0 +1,28 @@
+package org.jetbrains.kotlinx.dataframe.documentation
+
+import org.jetbrains.kotlinx.dataframe.api.*
+
+/**
+ * ## Column Expression
+ * In many DSLs, the lambda `expr {}` can be used to
```

Comment:
Sometimes it's "expr("name") { }`. And look at my comment below

Reply:
yes, but not everywhere. This is the common docs part, so if a version of `expr` supports a name, I'll mention it at the specific version.